test/index.js

```diff
@@ -33,25 +33,52 @@ function Just(x) {
   return {constructor: MaybeTypeRep, isNothing: false, isJust: true, value: x};
 }
 
+function mock($$type) {
+  return {constructor: {'@@type': $$type}};
+}
+
+function parsed(namespace, name, version) {
+  return {namespace: namespace, name: name, version: version};
+}
```

Comment:
Let's add `--rule 'max-len: [off]'` to the ESLint options for __test/index.js__.